esdt-safe/src/lib.rs

```diff
@@ -41,15 +42,10 @@ pub trait EsdtSafe:
         self.first_batch_id().set_if_empty(1);
```

Comment:
I think `first_batch_id` should be renamed with `current_batch_id`

Reply:
Eh, maybe, but not for now. Let's leave it as it is.